File: backend/app/database/supabaseClient.js
import {createClient} from '@supabase/supabase-js';

const supabaseUrl = backend.env.SUPABASE_URL;
const supabaseAnonKey = backend.env.SUPABASE_ANON_KEY;

export const supabase = createClient(supabaseUrl, supabaseAnonKey);

File: backend/app/database/supabase_client.py
import os
from supabase import create_client, Client
from dotenv import load_dotenv

load_dotenv()

url: str = os.environ.get("SUPABASE_URL")
key: str = os.environ.get("SUPABASE_KEY")

supabase: Client = None
if url and key:
    try:
        supabase = create_client(url, key)
        print("Supabase client created successfully!")
    except Exception as e:
        print(f"Failed to create Supabase client: {e}")
        supabase = None
else:
    print("Cannot create Supabase client: Missing URL or key")